### DISCUSSION TITLE
Where is GitLens

### DISCUSSION THREAD

**Discussion by driskell** [gitkraken/vscode-gitlens#2314] [Q&A] [answered]
This is 99% me not seeing something.

It seems my VSCode updated and now when I click GitLens in the sidebar I get "What's New" and a set of information about what's new. I can't for the life of me find where to actually use GitLens. It used to display information and commits and working changes. I managed to get to some Get Started place where I clicked things and ticks appeared, and I realised only at that time you can tick things away. Marked those done hoping it was an onboarding prison. But yeh still stuck on What's New 🤣 I clearly must need to do something somewhere or is it all moved and the GitLens icon is now the What's New?

**Comment by driskell** [comment, marked as answer]
Haha OK education has happened somewhere else. That pane is intended to be a support/features pane and everything is integrated elsewhere, and there's the commit tab I had somehow hidden for whatever reason and confused the hell out of myself. :+1:

**Comment by d13** [comment]
Glad you're up and running! By default, our views are under the Source Control side bar. 

If you find you still prefer the GitLens views under the GitLens side bar, you can run "GitLens: Welcome (Quick Setup)" from the command palette and select "GitLens Layout" in the Views section.

![Screen Shot 2022-11-03 at 10 26 35 AM](https://user-images.githubusercontent.com/293282/199747356-cf011a1e-5f5c-4cb9-8d4f-a159e513196d.png)

**Reply by driskell** [reply]
> Glad you're up and running! By default, our views are under the Source Control side bar.

Thanks - I did find that now. It was just extremely irregular for me initially, as every single sidebar tab I have ever used in VSCode contains the corresponding functionality for that extension. And it seems GitLens uses it to display "content" about the extension, and not display any of the functionality, which is very unusual. Perhaps it just needs an introduction "Ready to start? You'll find everything on the Source (Icon-here-and-link) panel. Come back here for news and updates and the best guides and tutorials"

I would also suggest the tickboxes be square like the rest of the system. I never knew initially you could tick the blue circles! Even VSCode itself uses a completely different look for a tickbox.
